docs/Validation-and-Serialization.md

```diff
@@ -301,6 +301,68 @@ In that case the function returned by `schemaCompiler` returns an object like:
 * `error`: filled with an instance of `Error` or a string that describes the validation error
 * `value`: the coerced value that passed the validation
 
+<a name="schema-resolver"></a>
+#### Schema Resolver
+
+The `schemaResolver` is a function that works togheter with the `schemaCompiler`: you can't use it
```

Comment:
```suggestion
The `schemaResolver` is a function that works together with the `schemaCompiler`: you can't use it
```

Reply:
@Eomm can you apply this so that we can land it and release in 2.9.0?